docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -104,6 +105,42 @@ This is why [selectors](selecting-page-elements/selectors.md) and
 [client functions](obtaining-data-from-the-client.md) need the test controller object when they are
 called from Node.js callbacks.
 
+### Setting Test Speed
+
+TestCafe allows you to specify the speed of test execution.
+
+By default, tests run at the maximum speed. However, if you need to watch a test running
+to understand what happens in it, this speed may seem too fast. In this instance, use the `t.setTestSpeed` method
+to slow the test down.
+
+```text
+t.setTestSpeed( factor )
+```
+
+Parameter  | Type      | Description
+---------- | --------- | -----------
+`factor`   | Number    | Specifies the test speed. Must be a number between `1` (the fastest) and `0.01` (the slowest).
+
+If speed is also specified for an [individual action](actions/action-options.md#basic-action-options), the action speed setting overrides test speed.
```

Comment:
I missed this. It should be changes to `override`, thanks.  /cc @VasilyStrelyaev